minetest.conf.example

```diff
@@ -9,6 +9,11 @@
 # 0 to disable
 #share_bones_time = 1200
 
+# How much earlier can the bones of a dead playerbe looted
+# by everyone if the player dies in a protected area they don't own.
+# 0 to disable. 
+# share_bones_time_early = share_bone_time / 4
```

Comment:
This should be set to 300, and the "default" of 1/4 moved to description. Also, remove the space between the `#` and the setting name.